docs/support/faq.md

```diff
@@ -118,6 +118,10 @@ Check out this PKP x WalletConnect example [here](https://github.com/LIT-Protoco
 
 Yes! Check out this [sample project.](https://github.com/dOrgJelli/lit-protocol-wasm-test/blob/d4b8873f9a5bceaf98e7f7a1bf325bf597cbfa40/src/App.js#L6-L51.)
 
+### 12. Why I'm getting following error message while making connection to Lit nodes "Error: Unable to verify the first certificate in nodejs"?
```

Comment:
Minor comment on sentence structure. Consider:

"I am getting the following error when connecting to the Lit nodes: "Error: Unable to verify the first certificate in nodejs". What may be causing this?"

Reply:
Nicely done @a1ttech!